components/cd-byline/cd-byline.css

```diff
@@ -0,0 +1,41 @@
+.cd-byline {
+  margin-top: 2rem;/* same as pseudo's padding-bottom */
+}
+
+/* The border is not on the container itself due to potential left/right padding
+ * values that can interfere. Ex: when it has the `content-width` class. */
+.cd-byline::after {
+  content: "";
+  display: block;
+  padding-bottom: 2rem;
+  border-bottom: 1px solid var(--cd-blue-grey);
+}
+
+.cd-byline__wrapper {
+  display: flex;
+}
+
+.cd-byline__image {
+  width: 4rem;
+  height: 4rem;
+  flex-grow: 0;
+  flex-shrink: 0;
+  flex-basis: 4rem;
```

Comment:
I am generally seeing flex shorthand in most components. Since this isn't using calc and won't trip up IE11 using shorthand, should it be converted?